includes/admin/settings/class-wcmpbe-settings-data.php

```diff
@@ -571,12 +571,12 @@ private function get_section_carrier_postnl_delivery_options(): array
                 "help_text" => __("Number of days you need to process an order.", "woocommerce-myparcelbe"),
             ],
             [
-                "name"      => WCMPBE_Settings::SETTING_CARRIER_DELIVERY_DAYS_WINDOW,
-                "condition" => WCMPBE_Settings::SETTING_CARRIER_DELIVERY_ENABLED,
-                "label"     => __("Show delivery date", "woocommerce-myparcelbe"),
-                "type"      => "toggle",
-                "default"   => self::ENABLED,
-                "help_text" => __("show_delivery_date_help_text", "woocommerce-myparcelbe"),
+                'name'      => WCMPBE_Settings::SETTING_CARRIER_ALLOW_SHOW_DELIVERY_DATE,
+                'condition' => WCMPBE_Settings::SETTING_CARRIER_DELIVERY_ENABLED,
+                'label'     => __('feature_allow_show_delivery_date_title', 'woocommerce-myparcel'),
+                'type'      => 'toggle',
+                'help_text' => __('feature_allow_show_delivery_date_help_text', 'woocommerce-myparcel'),
+                'default'   => self::ENABLED,
```

Comment:
Waarom is er de keuze gemaakt om `help_text` en `default` om te draaien?

Reply:
Geen bewuste keus